src/test/java/org/fundacionjala/coding/dennis/FizzBuzzTest.java

```diff
@@ -0,0 +1,87 @@
+package org.fundacionjala.coding.dennis;
+
+import org.junit.Before;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.List;
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ */
+public class FizzBuzzTest {
+
+    private List<StringBuilder> res;
+    private StringBuilder exp;
+
+    /**
+     *
+     */
+    @Before
+    public void ini() {
+        res = new ArrayList<>();
+        final int tam = 100;
+        res = FizzBuzz.listFizzBuzz(tam);
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testFizzBuzzDiv3() {
+        System.out.println(res);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)